synapseclient/api/__init__.py

```diff
@@ -30,6 +30,7 @@
 __all__ = [
     # annotations
     "set_annotations",
+    "set_annotations_async",
```

Comment:
I am also curious what is the purpose of "__all__" here and why did you add set_annotations_async?

Reply:
This is defining what can be imported from `synapseclient.api`:

https://www.geeksforgeeks.org/python-__all__/

It allows you to do:
`from synapseclient.api import set_annotations_async` instead of `from synapseclient.api.annotations import set_annotations_async`

Reply:
wow thank you for the explanation Bryan!